### PR TITLE
Inform developer that adapter property should be a class constructor

### PR DESCRIPTION
**Pull request by big-kahuna-burger**

### CODE REVIEW

File: lib/helpers/initialize_adapter.js [hunk 1]
```diff
@@ -7,6 +7,15 @@ module.exports = async function initializeAdapter(adapter = MemoryAdapter) {
   if (adapter === MemoryAdapter) {
     attention.warn('a quick start development-only MemoryAdapter is used');
   }
+  function isCtor() {
+    return !!adapter.prototype && !!adapter.prototype.constructor.name;
+  }
+
+  if (!isCtor()) {
+    throw new Error(
+      "Expected 'adapter' to be a constructor, provide a valid adapter in Provider config.",
```
Comment (panva) [inline lib/helpers/initialize_adapter.js]: Please swap the " and ', `'` as outer, `"` as inner around the word adapter.

File: lib/helpers/initialize_adapter.js [hunk 2]
```diff
@@ -7,6 +7,15 @@ module.exports = async function initializeAdapter(adapter = MemoryAdapter) {
   if (adapter === MemoryAdapter) {
     attention.warn('a quick start development-only MemoryAdapter is used');
   }
+  function isCtor() {
```
Comment (panva) [inline lib/helpers/initialize_adapter.js]: creating a function like this is not necessary, just use the expressions in the if.

File: lib/helpers/initialize_adapter.js [hunk 3]
```diff
@@ -7,6 +7,15 @@ module.exports = async function initializeAdapter(adapter = MemoryAdapter) {
...
+    return !!adapter.prototype && !!adapter.prototype.constructor.name;
```
Comment (panva) [inline lib/helpers/initialize_adapter.js]: no need to double `!` either of these wdyt?

**Comment by big-kahuna-burger** [comment]
see latest commit

**Comment by panva** [comment]
@big-kahuna-burger thank you for the contribution

**Pull request merged**